BookEcommerce/Controllers/VerifyAccountController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using BookEcommerce.Models.DTOs.Response.Base;
+using BookEcommerce.Services.Interfaces;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BookEcommerce.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class VerifyAccountController : ControllerBase
+    {
+        private readonly IVerifyAccountService verifyAccountService;
+        public VerifyAccountController(IVerifyAccountService verifyAccountService)
+        {
+            this.verifyAccountService = verifyAccountService;
+        }
+        [HttpPost("send-mail")]
+        public async Task<IActionResult> SendMail([FromForm] string Email)
+        {
+            await this.verifyAccountService.SendVerificationMail(Email);
```

Comment:
please take note that we need to verify any input before do any logic. 
And param name should be camel case

Reply:
Updated in email params validation